.github/CODEOWNERS

```diff
@@ -86,10 +86,10 @@
 
 # N-API
```

Comment:
Should we consider to rename in something like this:
`Node-API (ex N-API)`

Reply:
Do we need to reference the old name?
i.e. change to just
```suggestion
# Node-API
```
?